test/dynamo/test_subclasses.py

```diff
@@ -1007,6 +1014,17 @@ def forward(self):
             ],
         )
 
+    def test_wrapper_subclass_dynamo_attribute_access_on_intermediate(self):
+        def f(x_subclass):
+            tmp_subclass = torch.add(x, 1)
+            return torch.mul(tmp_subclass._scale, tmp_subclass._constant)
+
+        # should recompile for different scale size when dynamic=False
+        x = ScaledTensor(torch.randn(2, 4), torch.randn(3), constant=2)
+        out_ref = f(x)
+        out_test = torch.compile(f, backend="aot_eager")(x)
```

Comment:
dumbq, do you have to add full_graph=True to ensure that this isn't graphbreaking?